### PR TITLE
Hw4 task2 - count_dots_on_i function implementation and tests

### PR DESCRIPTION
**Pull request by IuliiaLanchuk**

### CODE REVIEW

File: homework4/task2/tests/test_mock_input.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from project2_mock_input.task2_mock_input import count_dots_on_i
+
+url = "https://example.com/"
```
Comment (Cjkjvfnby) [inline homework4/task2/tests/test_mock_input.py]: Please inline it.

**Review by Cjkjvfnby — APPROVED** [review]

File: homework4/task2/tests/test_mock_input.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from project2_mock_input.task2_mock_input import count_dots_on_i
+
+
+def test_count_dots_on_i():
```
Comment (Cjkjvfnby) [inline homework4/task2/tests/test_mock_input.py]: What does this test check?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: homework4/task2/tests/test_mock_input.py (hunk 2 above)
Comment (IuliiaLanchuk) [reply homework4/task2/tests/test_mock_input.py]: It is an example from task description (in docstring). This test check that in page https://example.com/ there are 59 'i' chars.

Comment (Cjkjvfnby) [reply homework4/task2/tests/test_mock_input.py]: Yes, it is an example of the function.  Could your check the requirements for the test?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: homework4/task2/tests/test_mock_input.py (hunk 2 above)
Comment (IuliiaLanchuk) [reply homework4/task2/tests/test_mock_input.py]: ok, there is info about test with Mock only.

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**